docs/deployment.rst

```diff
@@ -68,29 +68,26 @@ Our recommended GC tuning looks like this:
 Important Configuration Options
 -------------------------------
 
-The following logistical configurations should be changed for production environments. These options depend on your cluster
-layout.
+The following logistical configurations should be changed for production environments. These options depend on your cluster layout.
 
 Depending on how the |sr| instances coordinate to choose the :ref:`master<schemaregistry_single_master>`, you can deploy
-|sr| with |zk| (which can be shared with Kafka) or with Kafka itself. |zk|-based master election is available in all versions
-of |sr| and if you have an existing |sr| deployment you should continue to use it for compatibility. Kafka-based master election
-can be used in cases where |zk| is not available, for example for hosted or cloud Kafka environments, or if access to |zk|
-has been locked down.
+|sr| with |zk| (which can be shared with Kafka) or with Kafka itself.
 
-To configure the |sr| to use |zk| for master election, configure the ``kafkastore.connection.url`` setting.
+* Prior to version 4.0, |sr| master election was always |zk|-based. To configure the |sr| to use |zk| for master election, configure the ``kafkastore.connection.url`` setting.
```

Comment:
Since these docs are versioned (and this commit is to 4.1.0-post), why mention 4.0? Minimally, I think this should be not be the first sentence of this bullet (the important action is the 2nd sentence).